ogticrd/covid-safe-paths
# Fix report go back

#### Description:
Fixed the first screen in report so it doesn't have an goBack Arrow, depending from where you are accessing it.


#### Linked issues:



#### Screenshots:
![Simulator Screen Shot - iPhone 11 - 2020-07-02 at 12 25 12](https://user-images.githubusercontent.com/56127045/86386046-3aa49100-bc5f-11ea-8c64-1cf372996bd2.png)



#### How to test:
Press "Report" in bottomNavigation, it shouldn't have the goBack Button.
Press Settings > Report that I'm positive. I't should have the goback button.
